### DISCUSSION TITLE
Does a 'cd' command work in cosmos?

### DISCUSSION THREAD

**Discussion by greenec03** [CosmosOS/Cosmos#1881] [Q&A]
I'm wondering if a type of command that will put me into another directory work with the cosmos file structure since I'm finding it hard to find my answer so I might as well ask the dev team.

**Comment by raxracks** [comment]
store path as variable, create a cd command, modify path accordingly, use path as prefix for all io operations

**Comment by ghost** [comment]
Yes, it does! You can use Directory.SetCurrentDirectory. If it doesn't work ask it to be plugged or smth.

**Reply by greenec03** [reply]
it needs to be plugged lol, visual studio says its not a command
